### PR TITLE
feat(fast-development-site-react): add enums for tabType for better type safety and clarity

### PR DESCRIPTION
**Pull request by nalogiudice**
Closes [#555](https://github.com/Microsoft/fast-dna/issues/555)
<body>

<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-development-site-react/src/components/site/dev-tools.tsx [hunk 1]
```diff
@@ -10,7 +10,11 @@ export enum FrameworkEnum {
     angular = "angular"
 }
 
-export type tabType = "Code" | "Properties" | "Schema";
+export enum TabTypeEnum {
```
Comment (chrisdholt) [inline packages/fast-development-site-react/src/components/site/dev-tools.tsx]: I know that we have it on the "FrameworkEnum" above, but I think that's actually a mistake. Adding enum to a typed "enum" seems totally redundant to me. I'm sure you were following the existing pattern, but I don't know that it's the pattern we want.

**Comment by chrisdholt** [comment]
@nalogiudice updating this label to `pr : feature`

File: packages/fast-development-site-react/src/components/site/dev-tools.tsx (hunk 1 above)
Comment (nalogiudice) [reply packages/fast-development-site-react/src/components/site/dev-tools.tsx]: Sure, I can remove the word Enum - I can see how it is similar to saying  "ATM machine" :)

**Comment by nalogiudice** [comment]
@chrisdholt both enums updated.

**Pull request merged**